Resources/README.md

```diff
@@ -10,7 +10,8 @@
 * ### [WordPress Online Portfolio Tutorial](https://github.com/CIS320-team-3/CIS320-Team-3/blob/main/Resources/Tutorials/WordPress_for_Beginners.md)
   * This goes over the basics of WordPress and how to make a free website using it. With a focus on how to create portfolios using it, this is a valuable resource to check and use.
 
-* ### [How to Create and Host a Webpage on Github with Github Pages](https://github.com/CIS320-team-3/CIS320-Team-3/blob/main/Resources/Tutorials/Create_and_Host_Webpage_on_Github.md)
+* ### [How to Create and Host a Webpage on Github with Github Pages]()
```

Comment:
This looks better, one last suggestion, when looking at the file in preview mode there is an extra bullet point, and although the header's link is empty it still shows it as a link.

I'd recommending changing this code:
```
* ### [How to Create and Host a Webpage on Github with Github Pages]()
* 
  * [For those unfamiliar with Github or Github Pages, this is a step-by-step tutorial on how to create and host your webpage on the site.](https://youtu.be/2MsN8gpT6jY)
```
To something like this:
```
* ### How to Create and Host a Webpage on Github with Github Pages
  * [For those unfamiliar with Github or Github Pages, this is a step-by-step tutorial on how to create and host your webpage on the site.](https://youtu.be/2MsN8gpT6jY)
```

This removes the extra bullet point and doesn't make the header look like there is a link even though there isn't 😄